helpers.json

```diff
@@ -546,6 +546,38 @@
             "This helper can be used on strats with a shorter than 40 tile runway, if Samus just needs run through an enemy or speed blocks (as low as 32 tiles is expected in vanilla)."
           ]
         },
+        {
+          "name": "h_shinechargeSlideTemporaryBlue",
+          "requires": [
+            "canTemporaryBlue",
+            "canShinechargeMovementTricky"
+          ],
+          "devNote": [
+            "This represents sliding off the ledge while gaining a shinecharge, which leaves Samus falling with temporary blue."
+          ]
+        },
+        {
+          "name": "h_waterGetBlueSpeed",
```

Comment:
Only useful in Botwoon Etank I believe.  The aqueduct cases still lose their flash suit. Its fine to have around.  `h_stutterWaterGetBlueSpeed` has some extra uses in aqueduct that don't lose the flash suit.